buildSrc/src/main/java/org/elasticsearch/gradle/EmptyDirTask.java

```diff
@@ -49,11 +50,16 @@ public Chmod getChmod() {
         throw new UnsupportedOperationException();
     }
 
-    @Input
+    @Internal
```

Comment:
I'm assuming a non existing `File` can't be input ?

Reply:
The issue here was Gradle was complaining that we were using `@Input` for `@File` when we should have been using `@InputFile` or `@InputDirectory`. Using `@Input` tracks the `File` _path_, not it's contents, which is actually the behavior we want, but Gradle is issuing a warning here because we aren't being explicit. That's why I've added the input for the directory path, which is all we care about.